lib/plugins/aws/package/compile/events/apiGateway/lib/method/index.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
       const requestParameters = {};
       if (event.http.request && event.http.request.parameters) {
         Object.entries(event.http.request.parameters).forEach(([key, value]) => {
-          requestParameters[key] = value.required === undefined ? value : value.required;
+          requestParameters[key] = _.isObject(value) ? value.required || false : value;
```

Comment:
I think intention was to make it _required_ by default when object form is used. So I believe it should be:

```javascript
_.isObject(value)
  ? ((value.required != null) ? value.required : true)
  : value
```